rdt/transformers/numerical.py

```diff
@@ -285,13 +285,17 @@ def _get_distributions():
             raise
 
         return {
-            'gaussian': univariate.GaussianUnivariate,
+            'norm': univariate.GaussianUnivariate,
             'gamma': univariate.GammaUnivariate,
             'beta': univariate.BetaUnivariate,
-            'student_t': univariate.StudentTUnivariate,
+            't': univariate.StudentTUnivariate,
             'gaussian_kde': univariate.GaussianKDE,
-            'truncated_gaussian': univariate.TruncatedGaussian,
+            'truncnorm': univariate.TruncatedGaussian,
             'uniform': univariate.UniformUnivariate,
+            # the following are deprecated
+            'gaussian': univariate.GaussianUnivariate,
+            'student_t': univariate.StudentTUnivariate,
+            'truncated_gaussian': univariate.TruncatedGaussian,
```

Comment:
if in the init we replaced the deprecated distribution string with the new one before calling this method we could remove the deprecated mappings from this dictionary

Reply:
That's a good suggestion, thanks.